src/app/screens/Calculator/components/Button/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {View, Text, TouchableHighlight} from 'react-native';
+import styles from './styles.js';
+
+const Button = ({button}) => {
+  return (
+    <TouchableHighlight
+      activeOpacity={0.45}
+      underlayColor="#4bcaf9"
+      onPress={button.action}
```

Comment:
Prefer using props to maintain the original API. In this case, action should be onPress.

Reply:
Ready, I already changed it.